northstar/src/runtime/mod.rs

```diff
@@ -60,10 +60,33 @@ pub(self) type RepositoryId = String;
 pub use api::container::*;
 pub(self) use repository::Repository;
 pub(self) use state::MountedContainer;
-
 pub(self) type ExitCode = i32;
 pub(self) type Pid = u32;
 
+#[async_trait]
+pub(self) trait Process {
+    fn pid(&self) -> Pid;
+    async fn start(&mut self) -> Result<(), Error>;
+    async fn stop(mut self, timeout: time::Duration) -> Result<ExitStatus, Error>;
+}
+
+#[async_trait]
+pub(self) trait Launcher {
```

Comment:
will probably need different visibility if we implement it in another module/crate

Reply:
The `Launcher` is not publicly exchangeable by any user of `northstar::runtime`. This is not intended and would need to export a lot of more stuff we probably don't want to export.